src/OpenLoco/src/CommandLine.cpp

```diff
@@ -389,6 +409,7 @@ namespace OpenLoco
         std::cout << "                  - info, warning, error, verbose, all" << std::endl;
         std::cout << "                  Example: --log_levels \"all, -verbose\", logs all but verbose levels" << std::endl;
         std::cout << "                  Default: \"info, warning, error\"" << std::endl;
+        std::cout << "--all      -a     For compare, print out all divergences";
```

Comment:
This is missing a linebreak at the end:
```suggestion
        std::cout << "--all      -a     For compare, print out all divergences" << std::endl;
```